www/src/components/blog-post-preview-item.js

```diff
@@ -26,7 +26,7 @@ class BlogPostPreviewItem extends React.Component {
           }}
         >
           <img
-            alt={`Avatar for ${post.frontmatter.author.id}`}
+            alt=""
```

Comment:
This removes the existing alt text and replaces it with a blank string - was that desired?

Reply:
Providing content here is not really beneficial, as the exact same string (name of the author) is provided shortly after, so that content would be read out twice by screen readers. See [this guide](https://webaim.org/techniques/alttext/#context)

Reply:
Thanks for being cautious :)

Reply:
Thanks for being informative! I didn't realise that unnecessary alt text could actually be damaging.